frontend/package.json

```diff
@@ -16,7 +16,7 @@
     "dependencies": {
         "@headlessui/react": "^1.7.8",
         "@heroicons/react": "^2.0.14",
-        "@mui/base": "^5.0.0-alpha.116",
+        "@mui/base": "5.0.0-alpha.116",
```

Comment:
The newer release from `@mui/base` is breaking a lot of our components (breaking changes to export patterns). I just wanted to make sure we keep using the working version (until the components are adjusted in a different PR) even when using `npm i` instead of `ci`. Sometimes, at least I do still end up in a scenario where I need a fresh install and do not want to get into issues with our components. I plan on adjusting the components soon, then this lock can be removed.

Reply:
Sounds good 🙂